src/components/common/ServicesTable/ServiceTable.js

```diff
@@ -114,10 +114,10 @@ const RecipientTable = ({
               },
             ]}
           >
-            <Input defaultValue={record.recipient_id} />
+            <Input defaultValue={record.recipient.first_name} />
```

Comment:
Using .recipient is strange as I believe that pulls directly from recipient data in the redux store, but let's merge this and compare a new POST for services to the data to see if it matches.